backend/tests/conftest.py

```
@@ -16,20 +17,23 @@
 # Create tables in the test database
 Base.metadata.create_all(bind=engine)
 
+
 @pytest.fixture(scope="function")
```

Ruff (0.8.2)

`scope='function'` is implied in `@pytest.fixture()`

Remove implied `scope` argument

(PT003)